StripeFinancialConnections/StripeFinancialConnectionsTests/ConsentPaneSnapshotTests.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ConsentPaneSnapshotTests.swift
+//  StripeFinancialConnections
+//
+//  Created by Mat Schmid on 2024-07-04.
+//
+
+import UIKit
+import XCTest
+
+@_spi(STP) import StripeCoreTestUtils
+@testable import StripeFinancialConnections
+
+class ConsentPaneSnapshotTests: STPSnapshotTestCase {
+    private var fcs: FinancialConnectionsSheet!
+
+    override func setUp() {
+        super.setUp()
+
+        self.fcs = FinancialConnectionsSheet(
+            financialConnectionsSessionClientSecret: "clientSecret",
+            returnURL: "returnURL"
+        )
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        fcs = nil
+    }
+
+    func testConsentPane() {
+        presentAndLoadFcs()
```

Comment:
naming is subjective and its 100% non-blocking, but am curious on your thoughts of why prefer to use abbreviations vs. typing the whole name out?

so lets say there's a someone just reading the code and they see

A:

```
func testConsentPane() {
   presentAndLoadFcs()
```

B: 

```
func testConsentPane() {
   presentAndLoadFinancialConnectionSheet()
```

(Similar story is `fcs = nil` variable vs. `financialConnectionSheet = nil`)

What would be the key arguments why (A) beats (B) overall?

Reply:
I'll change this to B!